packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/project/services/RepresentationExportData.java

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.application.project.services;
+
+import java.util.UUID;
+
+/**
+ * Used to persist representation data in the exported project.
+ *
+ * @author sbegaudeau
+ */
+public record RepresentationExportData(
+        UUID id,
+        UUID projectId,
+        String descriptionId,
+        String targetObjectId,
+        String label,
+        String kind,
+        String content
```

Comment:
Shouldn't this include the migrationVersion et lastMigrationPerformed to be able to make sense of the (possibly old) data in a future import?